x/ethbridge/client/cli/tx.go

```diff
@@ -75,6 +78,17 @@ func GetCmdBurn(cdc *codec.Codec) *cobra.Command {
 
 			txBldr := authtxb.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
 
+			ethereumChainID, err := strconv.Atoi(viper.GetString("ethereum-chain-id"))
+			if err != nil {
+				return err
+			}
+
+			tokenContractString := viper.GetString("token-contract-address")
```

Comment:
use `const` for the flag

Reply:
https://github.com/cosmos/peggy/commit/7e1201a2ec806b75a086bced38f059101798adb5